### ISSUE TITLE
Change default Light Level config value for 1.18

### ISSUE THREAD

**Issue by Rafii2198** [shedaniel/LightOverlay-Fabric#128]
As of 1.18, mobs need complete darkness to spawn. It would be nice if the default option from version 1.18 would be set to 0 because now if people won't change it themselves it will spread misinformation.

**Comment by neilwightman** [comment]
Install modmenu too and you can configure the X to only appear on level 0.  Im using it on fabric.   I had to change yellow X to red too, but its working well.
https://www.curseforge.com/minecraft/mc-mods/modmenu

![image](https://user-images.githubusercontent.com/731426/144846341-00d7c419-1d7a-4227-8640-9e076bf3feb8.png)

**Comment by Rafii2198** [comment]
I know, I already did it, but my point is that it shouldn't be necessity.

Like, is right for an informative mod to show wrong information by default? 

And just because it is possible for change shouldn't mean that every user needs to also download another mod if they don't have it already and change that value manually, some people don't know that this mod shows wrong information and there are others who doesn't know that it was changed in Minecraft at all etc.
